frontend/src/app/supportAdmin/TenantDataAccess/TenantDataAccessFormContainer.tsx

```diff
@@ -45,14 +44,10 @@ const TenantDataAccessFormContainer = () => {
         justification: justification,
       },
     }).then(() => {
-      const alert = (
-        <Alert
-          type="success"
-          title="Tenant Data Access Updated"
-          body="You now have access to tenant data for the requested organization."
-        />
+      showSuccess(
```

Comment:
⚠️ These success toast notifications aren't actually visible to the users because on `line 56` we `setSubmitted(true)` which `return <Navigate to="/reload-app" />;` on `line 61` so we are redirected to `/` and that toast notification is lost. We should address this pattern used throughout the support admin workflow to display feedback to our users.